source/module_base/kernels/math_op.cpp

```diff
@@ -1,11 +1,14 @@
 #include "module_base/kernels/math_op.h"
+
 #include "module_base/libm/libm.h"
 
+#include <cmath>
+
 namespace ModuleBase {
 
 template <typename FPTYPE>
 __inline__
-FPTYPE __fact(const int n) {
+FPTYPE _fact(const int n) {
```

Comment:
is there any special reason to change the function name from __ to _

Reply:
The C++ standard reserves names starting with __.